speech/infinite_streaming_from_file/infinite_audio.go

```diff
@@ -0,0 +1,236 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command infinite_audio pipes the stdin audio data to
+// Google Speech API and outputs the transcript.
+
+// Before executing this program, the sampleRate parameter needs to be adjusted
+// to match your audio file, 16000 is used in this program as an example
+
+// To run this program, you could use the following command:
+// cat [your_audio_filename].wav | go run infinite_audio.go
+
+package main
```

Comment:
Please update the comment to be a "package comment": https://github.com/golang/go/wiki/CodeReviewComments#package-comments

```
// The infinite_streaming_from_file command pipes ...
// ...
package main
```

Reply:
Will you consolidate the comment with the comments above? No need to have both.

Reply:
I have already added the package comment just above package main between /* and */ 
Did you see that? Long sentence package comment needs to be split into several lines.

Reply:
Yes, I saw that. Please make it all one comment. If you need multiple lines in the package comment, you can add a newline in the block comment (`/* ... */`).